keystore/keystore.go

```diff
@@ -72,12 +74,65 @@ func (ks *Keystore) New(auth, label string, valAddr bool) (*KeyData, error) {
 	return kd, nil
 }
 
-func (ks *Keystore) Delete(addrress crypto.Address, auth string) error {
+func (ks *Keystore) Unlock(auth, indexOrAddress string) error {
+	// get the keydata struct
+	keyObject, err := ks.getKeyObject(indexOrAddress)
+	if err != nil {
+		return err
+	}
+
+	// do decrypt and return the data
+	key, err := key.DecryptKeyFile(keyObject.Filename, auth)
+	if err != nil {
+		return fmt.Errorf("Decryption process fail")
```

Comment:
1- For warning/Error/Success messages use helper methods. look [`cmd/utils.go`](https://github.com/gallactic/gallactic/blob/d0947abb91a59569d6a18e152f52be2ce32e7693/cmd/utils.go#L165)
2- Better to report error message to the user. Look [here](https://github.com/gallactic/gallactic/blob/d0947abb91a59569d6a18e152f52be2ce32e7693/cmd/gallactic/key/inspect.go#L37).

Reply:
resolved  in [commit](https://github.com/gallactic/gallactic/pull/76/commits/91e618e9ee06fb2418c8cdbc25f9e69f530b6c14)